packages/test-utils/package.json

```diff
@@ -42,11 +42,23 @@
   },
   "peerDependencies": {
     "@testing-library/react": "^14.0.0",
-    "react": "18.2.0"
+    "react": "18.2.0",
+    "react-dom": "18.2.0"
+  },
+  "peerDependenciesMeta": {
```

Comment:
test-utils contains both Jest configuration and test utilities. React related dependencies are only needed to use the React test utilities. If a package is only using the Jest configuration, it doesn't need these.

https://docs.npmjs.com/cli/v7/configuring-npm/package-json#peerdependenciesmeta